src/Raven.Server/Documents/Indexes/MapReduce/OutputToCollection/OutputReduceToCollectionActions.cs

```diff
@@ -197,6 +199,18 @@ public bool DeleteDocuments(IndexingStatsScope stats, TransactionOperationContex
                             {
                                 enqueue.GetAwaiter().GetResult();
                             }
+                            catch (TaskCanceledException)
```

Comment:
Any reason to catch `TaskCanceledException` instead of `OperationCancelledException`? Note that `OperationCancelledException` is base.

Also we have the catch in `OutputReduceIndexWriteOperation`. Can we change it on both?

Reply:
Because `GetResult` will throw `TaskCanceledException`.

Reply:
done